### PR TITLE
#761 contact administrator via'zvorotniy zvyazok'page1

### PR DESCRIPTION
**Pull request by ohorodnykostap**
Implementation of a feedback form for users and forum visitors.

### CODE REVIEW

File: FrontEnd/src/components/Contact/Contact.jsx [hunk 1]
```diff
@@ -1,28 +1,202 @@
...
+                </div>
+            )}
+        </div>
+    );
 };
 
 export default Contact;
+
+Contact.propTypes = {
+    currentFormNameHandler: PropTypes.func,
```
Comment (YanZhylavy) [inline FrontEnd/src/components/Contact/Contact.jsx:201]: Does this component have that prop?

File: FrontEnd/src/components/Contact/Contact.module.css [hunk 2]
```diff
@@ -72,27 +92,211 @@
...
+}
+
+.modal_feedback_content {
+    background-color: white;
+    padding: 20px;
+    position: relative;
+    width: 550px;
+    height: 150px;
+    text-align: center;
+    display: flex; /* Включаємо Flexbox */
```
Comment (YanZhylavy) [inline FrontEnd/src/components/Contact/Contact.module.css:268]: I'm not sure these comments are appropriate.

File: FrontEnd/src/components/Contact/text.js [hunk 3]
```diff
@@ -14,170 +14,8 @@ const contactText = {
             level: 4,
             text: 'What is Kickstarter\'s copyright policy?'
```
Comment (YanZhylavy) [inline FrontEnd/src/components/Contact/text.js:15]: Why did you leave it?

File: FrontEnd/src/components/Contact/Contact.module.css (hunk 2 above)
Comment (ohorodnykostap) [reply FrontEnd/src/components/Contact/Contact.module.css:268]: Done.

File: FrontEnd/src/components/Contact/text.js (hunk 3 above)
Comment (ohorodnykostap) [reply FrontEnd/src/components/Contact/text.js]: Done.

**Pull request closed without merging**